include/spm/dispdrv.h

```diff
@@ -46,7 +46,7 @@ void dispReInit();
 /*
     Register a function to be displayed this frame
 */
-void dispEntry(s8 cameraId, s8 renderMode, f32 z, DispCallback * callback, void * callbackParam);
+void dispEntry(f32 z, s8 cameraId, s8 renderMode, DispCallback * callback, void * callbackParam);
```

Comment:
Since floats and non-floats are passed in different orders, these function declarations both work. The spm-headers one made more sense, ghidra should be updated to reflect that instead

Reply:
alrighty